source/common/ratelimit/ratelimit_impl.h

```diff
@@ -72,7 +78,7 @@ class NullClientImpl : public Client {
   // RateLimit::Client
   void cancel() override {}
   void limit(RequestCallbacks& callbacks, const std::string&, const std::vector<Descriptor>&,
-             const Tracing::TransportContext&) override {
```

Comment:
is TransportContext used somewhere now?

Reply:
this is fine, we can delete TransportContext entirely and clean private filters on submodule update.